npm/cypress-schematic/src/builders/cypress/index.ts

```diff
@@ -9,14 +9,14 @@ import { asWindowsPath, normalize } from '@angular-devkit/core'
 import * as os from 'os'
 import { dirname, join } from 'path'
 import { open, run } from 'cypress'
-import { from, noop, Observable, of } from 'rxjs'
+import { forkJoin, from, noop, Observable, of } from 'rxjs'
 import { catchError, concatMap, first, map, switchMap, tap } from 'rxjs/operators'
 import { CypressBuilderOptions } from './cypressBuilderOptions'
 
 type CypressOptions = Partial<CypressCommandLine.CypressRunOptions> &
 Partial<CypressCommandLine.CypressOpenOptions>;
 
-type StartDevServerProps = {
+type CypressStartDevServerProps = {
   devServerTarget: string
   watch: boolean
   context: any
```

Comment:
Not sure why this is typed as `any`, can you try changing it to `BuilderContext` so that the `startDevServer` function has proper typings?